build.gradle

```diff
@@ -6,18 +6,18 @@ buildscript {
     'sourceCompatibility': JavaVersion.VERSION_1_7,
     'targetCompatibility': JavaVersion.VERSION_1_7,
     'okhttp': '3.10.0',
-    'supportLibrary': '28.0.0-alpha3'
+    'supportLibrary': '1.0.0-rc01'
   ]
 
   ext.deps = [
     androidPlugin: 'com.android.tools.build:gradle:3.2.0-beta02',
     okhttp: "com.squareup.okhttp3:okhttp:${versions.okhttp}",
     mockWebServer: "com.squareup.okhttp3:mockwebserver:${versions.okhttp}",
     pollexor: 'com.squareup:pollexor:2.0.4',
-    supportCompat: "com.android.support:support-compat:${versions.supportLibrary}",
-    supportFragment: "com.android.support:support-fragment:${versions.supportLibrary}",
-    supportAnnotations: "com.android.support:support-annotations:${versions.supportLibrary}",
-    supportExifInterface: "com.android.support:exifinterface:${versions.supportLibrary}",
+    supportCompat: "androidx.appcompat:appcompat:${versions.supportLibrary}",
+    supportFragment: "androidx.fragment:fragment:${versions.supportLibrary}",
```

Comment:
This is only for the sample or something, right?

Reply:
yeah.